### PR TITLE
Edit UG, DG, PPP

### PR DESCRIPTION
**Pull request by TSAI-HSIAO-HAN**

### CODE REVIEW

File: docs/team/TSAI-HSIAO-HAN.adoc [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+== PROJECT: Mark
+
+---
+
+== Overview
+This document will introduce my role and contribution in Mark.
+Mark is a 6 weeks group project for the module CS2103T Software Engineering.
+Our team composed of 5 NUS Computer Science year 2 students.
+We have a team meeting every Tuesday and Thursday.
```
Comment (openorclose) [inline docs/team/TSAI-HSIAO-HAN.adoc]: Just some comments on the PPP, is the team meeting needed?

File: docs/team/TSAI-HSIAO-HAN.adoc [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+From this project, we learn the skill of designing a Java application with basic CRUD structure,
+working on git,
+distributing works on Github,
+negotiating and teamwork.
+
+
+Mark is a __desktop bookmark manage__r used for organising websites.
```
Comment (openorclose) [inline docs/team/TSAI-HSIAO-HAN.adoc]:
```suggestion
Mark is a __desktop bookmark manager__ used for organising websites.
```

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -1012,28 +1054,116 @@ Format: `*reminder-edit* INDEX [t/TIME_DATE] [n/NOTE]`
...
+To delete a reminder:
+
+. The index of the reminder in the reminder list is 1.
+You type the command `*reminder-delete* 1` into the command box and press _Enter_ to execute it.
+
+image::DeleteReminderUi1.png[]
+//The image of dashboard that shows the reminder for Luminus.
+
+. The result box will show the message: +
+"Edited Reminder: Do the quiz and the assignment Time: 10/11/2019 0700 URL:https://luminus.nus.edu.sg/"
```
[the remark this answers is not in the document]
Comment (TSAI-HSIAO-HAN) [reply docs/UserGuide.adoc]: yes

File: docs/diagrams/ReminderAssociationDiagram.puml [hunk 4]
```diff
@@ -19,7 +19,7 @@ Class Folder
 Class UniqueBookmarkList
 }
 
-ReminderAssociation --> "*" Reminder
+ReminderAssociation -- "*" Reminder
```
Comment (TSAI-HSIAO-HAN) [inline docs/diagrams/ReminderAssociationDiagram.puml]:
```suggestion
ReminderAssociation --> "*" Reminder
```

File: docs/team/TSAI-HSIAO-HAN.adoc (hunk 1 above)
Comment (TSAI-HSIAO-HAN) [reply docs/team/TSAI-HSIAO-HAN.adoc]: Can mention.

File: docs/UserGuide.adoc [hunk 5]
```diff
@@ -1012,28 +1054,116 @@ Format: `*reminder-edit* INDEX [t/TIME_DATE] [n/NOTE]`
...
+****
+For example:
+
+You want to plan your time table for next semester, which is a task on reminder list.
+Now you can use the `reminder-goto` command to open the website.
+
+To open the bookmark of a reminder:
+
+. The index of the reminder in the reminder list is 3.
+You type the command `*reminder-goto* 2` into the command box and press _Enter_ to execute it.
```
[the remark this answers is not in the document]
Comment (TSAI-HSIAO-HAN) [reply docs/UserGuide.adoc]: both 2, I forgot to change

**Pull request merged**